optuna_dashboard/ts/components/TrialList.tsx

```diff
@@ -339,6 +341,14 @@ export const TrialList: FC<{ studyDetail: StudyDetail | null }> = ({
       (state) => allTrials.filter((t) => t.state === state).length
     )
   }, [studyDetail?.trials])
+  const listParentRef = React.useRef(null)
+
+  const rowVirtualizer = useVirtualizer({
+    count: trials.length,
+    getScrollElement: () => listParentRef.current,
+    estimateSize: () => 75,
+    overscan: 10,
```

Comment:
[question] How did you decide those numbers?

Reply:
I've changed the value for estimateSize to 73.31 based on the actual height.
<img width="335" alt="image" src="https://github.com/optuna/optuna-dashboard/assets/30489874/5ce5d5d8-48a8-4273-9393-7653f5fdade4">

On the other hand, the value for overscan was determined heuristically and I don't have a strong opinion.
Do you have any ideas on how to decide this value?